cli/command/trust/signer_remove.go

```diff
@@ -43,11 +43,9 @@ func removeSigner(cli command.Cli, options signerRemoveOptions) error {
 	var errRepos []string
 	for _, repo := range options.repos {
 		fmt.Fprintf(cli.Out(), "Removing signer \"%s\" from %s...\n", options.signer, repo)
-		if err := removeSingleSigner(cli, repo, options.signer, options.forceYes); err != nil {
+		if _, err := removeSingleSigner(cli, repo, options.signer, options.forceYes); err != nil {
```

Comment:
I would argue that there is no need to return a `bool` as all now that it's dropped here.

Reply:
I will implement more tests that need to know if the signer got actually removed, and I need the information through this boolean.